python/nutpie/compile_pymc.py

```diff
@@ -441,7 +470,7 @@ def extract_shared(x, user_data_):
         user_data = numba.carray(user_data_, (), record_dtype)
 
         _shared_tuple = shared_tuple
-        for index in literal_unroll(indices):
+        for index in numba.literal_unroll(indices):
```

Comment:
There was a numba issue some time ago where literal_unroll only did anything if it was used without the leading `numba.`. I'm not sure if that is still the case, maybe we should leave it just to be sure?

Reply:
Wow, that's crazy!!!

Done.

Reply:
Ref: https://github.com/numba/numba/issues/5344